gradio/processing_utils.py

```diff
@@ -126,7 +126,12 @@ def resize_and_crop(img, size, crop_type="center"):
 
 
 def audio_from_file(filename, crop_min=0, crop_max=100):
-    audio = AudioSegment.from_file(filename)
+    try:
+        audio = AudioSegment.from_file(filename)
+    except FileNotFoundError as e:
+        error_message = str(e)
+        if "ffprobe" in error_message:
+            print("Please install `ffmpeg` in your system to use Audio.")
```

Comment:
Maybe clarify ffmpeg is needed to use non-WAV file formats, not the Audio component in general.